api/handlers/node.go

```diff
@@ -38,7 +41,7 @@ type connectionJSON struct {
 	Direction string `json:"direction"`
 }
 
-type peerJSON struct {
+type PeerJSON struct {
```

Comment:
Unrelated to this PR, but if it already exposes this then let's give it a nice name, just `Peer` should be enough :)